javascript/javascript1/week2/main.js

```diff
@@ -0,0 +1,103 @@
+// Flight booking fullname function
+
+console.log(getFullName("khuram", "Ejaz", true));
+console.log(getFullName("john", "smith"));
+
+function getFullName(firstName1, lastName1, useFormalName) {
+  if (useFormalName == true && firstName1 !== "" && lastName1 !== "") {
```

Comment:
useFormalName will be analysed true or false based on the parameter passed, but its not required to use true/false.

ex: you can use if(useFormalName && firstName....)